build/squire-raw.js

```diff
@@ -1626,16 +1629,12 @@ var firstOrLastLine = function(self){
     var rect = range.getBoundingClientRect()
     var nodeOffset = rect.top
     var parentOffset = parentBlock.getBoundingClientRect().top
-    if(nodeOffset === parentOffset){
-      return {firstLine: true, lastLine: true}
-    }
     nodeOffset = nodeOffset - parentOffset
```

Comment:
although I'm not sure it is still relevant looking at the commit message sheds some light on its intention:

https://github.com/natejenkins/squire/commit/953ddbe49c06fac3986f8538535d482e7b43bfc3